templated_email/backends/vanilla_django.py

```diff
@@ -65,18 +65,28 @@ def _render_email(self, template_name, context,
                       template_dir=None, file_extension=None):
         response = {}
         errors = {}
-        prefixed_template_name = ''.join((template_dir or self.template_prefix, template_name))
         render_context = Context(context, autoescape=False)
+
         file_extension = file_extension or self.template_suffix
         if file_extension.startswith('.'):
             file_extension = file_extension[1:]
-        full_template_name = prefixed_template_name
-        if not prefixed_template_name.endswith('.%s' % file_extension):
-            full_template_name = '%s.%s' % (prefixed_template_name, file_extension)
+
+        # bastbnl allow for template selection (see #78). Requires django-render-block v0.3 or higher
+        if isinstance(template_name, (tuple, list, )):
+            prefixed_templates = template_name
+        else:
+            prefixed_templates = [template_name, ]
```

Comment:
I think that's a style preference, but I removed it anyway as I don't think we'll need to modify the style used within the project just because of my preferences ;-)